packages/runtime-dom/src/patchProp.ts

```diff
@@ -111,6 +111,13 @@ function shouldSetAsProp(
     return false
   }
 
+  // #13946 iframe.sandbox should always be set as attribute since setting
+  // the property to null results in 'null' string, and setting to empty string
+  // enables the most restrictive sandbox mode instead of no sandboxing.
+  if (key === 'sandbox') {
```

Comment:
I think we should be checking `el.tagName` too.

It isn't checked for `spellcheck`, `draggable`, `translate` and `autocorrect` because they are global and can apply to any element.

For other special `key` values we do usually check the `tagName`.

`form` is an outlier. It did originally check the `tagName`, but that check was removed because it applied to so many tags that the bloat wasn't worth it, see 180310cbe9fdc04137e5bc4ad2a7cbe89c212a0e.

I'm not entirely sure *why* we check the `tagName`. Presumably it's to reduce the risk of the workarounds leaking out and having unintended consequences.

Reply:
make sense